phpBB/phpbb/notification/manager.php

```diff
@@ -333,10 +333,34 @@ public function add_notifications_for_users($notification_type_name, $data, $not
 		foreach ($this->get_subscription_methods_instances() as $method)
 		{
 			$notified_users = $method->get_notified_users($notification_type_id, array('item_id' => $item_id));
+			$notification_method_name = $method->get_type();
+
 			foreach ($notified_users as $user => $notifications)
 			{
 				unset($notify_users[$user]);
 			}
+
+			/**
+			* Allow filtering the notify_users array by notification_type_name & notification_method_name for a notification that is about to be sent.
+			* Here, $notify_users is already filtered from users who've already been notified.
+			*
+			* @event core.notification_manager_add_notifications_for_users_modify_data
+			* @var	string	notification_type_name		The notification type identifier
+			* @var	string	notification_method_name	The notification method identifier (read only)
+			* @var	array 	data						Data specific for the notification_type_name used will be inserted
+			* @var	array 	notify_users				The array of userid that are going to be notified for this notification. Set to array() to cancel or unset each user by userid.
```

Comment:
Same as above. I think the original description from the docblock is clear enough and more on point.

Reply:
Can you please clarify this , what you want me to do here ???
I copied from https://github.com/phpbb/phpbb/pull/5862/files#diff-0d7ecc527ae7eeebc4f109c2fdfcb1abR282

Best regards 👍